linters/pre-commit-hooks/plugin.yaml

```diff
@@ -19,3 +19,148 @@ lint:
           success_codes: [0, 1]
           files:
             - ALL
+        - name: check-added-large-files
+          enabled: false
+          output: pass_fail
+          read_output_from: stdout
+          run: check-added-large-files ${target}
```

Comment:
Can we add `python3` to the front of these if appropriate? Shebangs don't work for these executions on Windows.

Reply:
Done

Reply:
False alarm. This was previously WAI, the files are .exe on Windows and inside of `Scripts/`, so no need for the python3 bit.

Reply:
Updated